llama-index-integrations/llms/llama-index-llms-sambanova/pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.llms.sambanova"
+
+[tool.llamahub.class_authors]
+SambanovaLLM = "Pradeep"
```

Comment:
I believe the class is called `Sambaverse`?

We should also add a line entry here for `SambaStudio` as well.

Reply:
Yes @nerdai, thanks for pointing this out. I updated this